ReactWindows/ChakraBridge/JsIndexedModulesUnbundle.cpp

```diff
@@ -0,0 +1,121 @@
+#include "pch.h"
+#include "ChakraHost.h"
+#include "JsIndexedModulesUnbundle.h"
+
+JsIndexedModulesUnbundle::JsIndexedModulesUnbundle(const wchar_t* sourcePath)
+{
+    this->sourcePath = sourcePath;
+}
+
+JsIndexedModulesUnbundle::~JsIndexedModulesUnbundle()
+{
+	if (fileHandle != NULL)
+	{
+		UnmapViewOfFile(byteBuffer);
+		CloseHandle(mapHandle);
```

Comment:
Since all handles are generated from the same LoadByteCode function, I think the same assumptions apply that if the file handle is not null, then we can close these in the destructor successfully.

---
In reply to: [103551314](https://github.com/ReactWindows/react-native-windows/pull/1017#discussion_r103551314) [](ancestors = 103551314,103548082)